include/verilogAST.hpp

```diff
@@ -748,17 +748,13 @@ class Always : public StructuralStatement {
       std::variant<std::unique_ptr<Identifier>, std::unique_ptr<PosEdge>,
                    std::unique_ptr<NegEdge>, std::unique_ptr<Star>>>
       sensitivity_list;
-  std::vector<std::variant<std::unique_ptr<BehavioralStatement>,
-                           std::unique_ptr<Declaration>>>
-      body;
+  std::vector<std::unique_ptr<BehavioralStatement>> body;
```

Comment:
why did this and the below switch from variants to BehavioralStatement only?

Reply:
Original code was a bug, probably a copy paste error from the module body variant type.  AFAIK you can't declare things inside an always, they have to be declared outside in the module body.